components/automate-ui/src/app/entities/jobs/job.effects.ts

```diff
@@ -37,9 +39,25 @@ export class JobEffects {
     mergeMap((action: JobActions) => this.requests.jobCreate(action.payload)),
     map((payload: {id: string}) => new JobCreateSuccess(payload)));
 
+  @Effect()
+  JobCreateSuccess$ = this.actions$.pipe(
+    ofType<JobCreateSuccess>(JobActionTypes.JOB_CREATE_SUCCESS),
+    map(() => new CreateNotification({
+      type: Type.info,
+      message: 'Created a scan job.'
```

Comment:
I believe this wording should be changed here, and similarly for updated and deleted below.
@jonong1972 can you confirm?
```suggestion
      message: 'Scan job created.'
```

Reply:
Sweet, I wasnt sure if we can pull that. But would like this. @vinay033 can update this?

Reply:
sure, I will update.